src/avt/Pipeline/Data/avtDataAttributes.C

```diff
@@ -2177,6 +2153,60 @@ avtDataAttributes::SetVariableDimension(int vd, const char *varname)
 }
 
 
+// ****************************************************************************
+//  Method: avtDataAttributes::SetVariableDimension
+//
+//  Purpose:
+//      Sets the variable dimension.
+//
+//  Arguments:
+//      vd       The new variable dimension.
+//
+//  Programmer: Hank Childs
+//  Creation:   September 4, 2001
+//
+//  Modifications:
+//    Kathleen Bonnell, Wed Oct  3 10:57:13 PDT 2001
+//    Add actualData, thisProcsActualData.
+//
+//    Hank Childs, Mon Feb 23 14:19:15 PST 2004
+//    Account for multiple variables.
+//
+//    Kathleen Bonnell, Thu Mar 11 10:32:04 PST 2004 
+//    DataExtents now always have dimension of 1. 
+//
+//    Kathleen Bonnell, Wed Mar 31 08:03:47 PST 2004
+//    Added a reason to the exception.
+//
+//    Hank Childs, Wed Dec  1 15:29:56 PST 2004
+//    Make sure varname is non-NULL, or we'll crash.
+//
+//    Jeremy Meredith, Thu Feb  7 17:52:59 EST 2008
+//    Added component extents for array variables.
+//
```

Comment:
Remove this line. One blank comment line is sufficient.

Reply:
The diff is confused; I didn't modify these lines. It is confused because I made a new variant of the setter up above and changed some things, so it thinks all of this is new. The only change to this code is calling the new setter at the bottom.

Reply:
Ok, it looks ok. Disregard my comment.